src/main/java/com/epam/ta/reportportal/core/dashboard/GetDashboardHandler.java

```diff
@@ -32,14 +32,24 @@
 public interface GetDashboardHandler {
 
 	/**
-	 * Get dashboard by id
+	 * Get dashboard on which use have {@link com.epam.ta.reportportal.auth.permissions.AclReadPermission} by id
 	 *
 	 * @param dashboardId    Dashboard id
 	 * @param projectDetails Project details
 	 * @param user           User
 	 * @return dashboard
 	 */
-	Dashboard getDashboard(Long dashboardId, ProjectDetails projectDetails, ReportPortalUser user);
+	Dashboard getPermittedDashboard(Long dashboardId, ProjectDetails projectDetails, ReportPortalUser user);
```

Comment:
let's remove a word Dashboard from here

Reply:
check if we really need projectDetails and user in params?

Reply:
removed